app/src/main/java/org/cru/godtools/activity/OnBoardingActivity.kt

```diff
@@ -0,0 +1,49 @@
+package org.cru.godtools.activity
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.viewpager.widget.ViewPager
+import org.cru.godtools.R
+import org.cru.godtools.adapter.OnboardingPagerAdapter
+
+class OnBoardingActivity : AppCompatActivity(), OnBoardingCallbacks {
+    private lateinit var viewPager: ViewPager
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_onboarding)
+        val pagerAdapter = OnboardingPagerAdapter(this, supportFragmentManager)
```

Comment:
Is `pagerAdapter` useful for anything? Why not put this after the `viewPager` instantiation and do `viewPager.adapter = OnboardingPagerAdapter(this, supportFragmentManager)`?